src/TorchSharp/Tensor/TensorExtensionMethods.cs

```diff
@@ -400,7 +395,7 @@ public static void Load(this Tensor tensor, System.IO.BinaryReader reader, bool
                 totalSize *= loadedShape[i];
             }
 
-            if (!skip && !loadedShape.SequenceEqual(tensor.shape))
+            if (!loadedShape.SequenceEqual(tensor.shape))
                 // We only care about this if the bytes will be written to the tensor.
```

Comment:
I don't understand why the shapes have to match for a skipped tensor. I get that we have to read past it, but that is just about advancing the stream pointer, so the saved size/shape information is sufficient.

Reply:
The reason is because the skip flag is no longer there.

Reply:
Okay, but by removing an argument, you just introduced a breaking change, right?

Reply:
Correct. 
Should I revert this part of the change?

Reply:
Unless there's a very compelling reason for breaking the API, or there is evidence that the old API could not possibly have been working correctly, we try to avoid doing so.

Reply:
Ok, I'll fix it :)